python/observability-online-eval/langgraph/computer-use-agent/main.py

```
@@ -0,0 +1,180 @@
+            ),
+        },
+        {
+            "role": "user",
+            "content": (
+                "find today's top 1 song on billboard's charts."
+            ),
+        },
+    ]
+    resp = await handle(messages)
+
+    with open("updates.json", "w") as f:
```

Ruff (0.11.9)

Async functions should not open files with blocking methods like `open`

(ASYNC230)